api/utils/keys/policy_test.go

```diff
@@ -33,23 +33,28 @@ var (
 		keys.PrivateKeyPolicyHardwareKeyTouch,
 		keys.PrivateKeyPolicyHardwareKeyPIN,
 		keys.PrivateKeyPolicyHardwareKeyTouchAndPIN,
+		keys.PrivateKeyPolicyWebSession,
 	}
 	hardwareKeyPolicies = []keys.PrivateKeyPolicy{
 		keys.PrivateKeyPolicyHardwareKey,
 		keys.PrivateKeyPolicyHardwareKeyTouch,
 		keys.PrivateKeyPolicyHardwareKeyPIN,
 		keys.PrivateKeyPolicyHardwareKeyTouchAndPIN,
+		keys.PrivateKeyPolicyWebSession,
 	}
 	hardwareKeyTouchPolicies = []keys.PrivateKeyPolicy{
 		keys.PrivateKeyPolicyHardwareKeyTouch,
 		keys.PrivateKeyPolicyHardwareKeyTouchAndPIN,
+		keys.PrivateKeyPolicyWebSession,
 	}
 	hardwareKeyPINPolicies = []keys.PrivateKeyPolicy{
 		keys.PrivateKeyPolicyHardwareKeyPIN,
 		keys.PrivateKeyPolicyHardwareKeyTouchAndPIN,
+		keys.PrivateKeyPolicyWebSession,
 	}
 	hardwareKeyTouchAndPINPolicies = []keys.PrivateKeyPolicy{
 		keys.PrivateKeyPolicyHardwareKeyTouchAndPIN,
+		keys.PrivateKeyPolicyWebSession,
 	}
 )
 
```

Comment:
The test `touch policy and pin policy` below is failing.  It is getting `web_session` as the response, and it's not immediately obvious to me why.

Reply:
The issue was that the "role set" had `web_session` as one of the required key policies. This isn't technically possible in the current setup, but this requirement would only be met by a key with the `web_session` policy. 

Note: If we ever wanted to require clusters to only be accessible through the WebUI, adding a `private_key_policy: web_session` option would be an easy way to do it.

I fixed the test by removing the `web_session` policy from the role set list.